src/writeData/subscriptions/components/SubscriptionDetailsPage.tsx

```diff
@@ -0,0 +1,203 @@
+// Libraries
+import React, {FC, useState, useContext, useEffect} from 'react'
+import {useSelector} from 'react-redux'
+import {useParams} from 'react-router-dom'
+
+// Components
+import {
+  Page,
+  FlexBox,
+  JustifyContent,
+  AlignItems,
+  SpinnerContainer,
+  TechnoSpinner,
+  IconFont,
+  Heading,
+  HeadingElement,
+  FontWeight,
+  SubwayNav,
+  SubwayNavModel,
+} from '@influxdata/clockface'
+import BrokerDetails from 'src/writeData/subscriptions/components/BrokerDetails'
+import ParsingDetails from 'src/writeData/subscriptions/components/ParsingDetails'
+import SubscriptionDetails from 'src/writeData/subscriptions/components/SubscriptionDetails'
+import CloudUpgradeButton from 'src/shared/components/CloudUpgradeButton'
+import GetResources from 'src/resources/components/GetResources'
+
+// Contexts
+import {
+  SubscriptionUpdateProvider,
+  SubscriptionUpdateContext,
+} from 'src/writeData/subscriptions/context/subscription.update'
+import {WriteDataDetailsContext} from 'src/writeData/components/WriteDataDetailsContext'
+import WriteDataDetailsProvider from 'src/writeData/components/WriteDataDetailsContext'
+
+// Types
+import {AppState, ResourceType, Bucket} from 'src/types'
+
+// Utils
+import {getAll} from 'src/resources/selectors'
+
+// Actions
+import {shouldShowUpgradeButton} from 'src/me/selectors'
+
+// Graphics
+import {FormLogo} from 'src/writeData/subscriptions/graphics/FormLogo'
+
+// Styles
+import 'src/writeData/subscriptions/components/SubscriptionDetailsPage.scss'
+
+import {
+  SubscriptionListContext,
+  SubscriptionListProvider,
+} from 'src/writeData/subscriptions/context/subscription.list'
+
+interface SubscriptionNavigationModel extends SubwayNavModel {
```

Comment:
now using the reusable component. however there will need to be tweaks in the next round to the component to match the design - handing in a header when it's on update and icons not turning into a tick when it's no longer the create.